Packs/SlackITAdmin/Integrations/Slack_IT_Admin/Slack_IT_Admin.py

```diff
@@ -0,0 +1,724 @@
+        self.active = active
+        self.iden = iden
+        self.username = username
+        self.email = email
+        self.errorCode = errorCode
+        self.errorMessage = errorMessage
+        self.details = details
+        self.data = {
+            "brand": self.brand,
+            "instanceName": self.instanceName,
+            "success": success,
+            "active": active,
+            "id": iden,
+            "username": username,
+            "email": email,
+            "errorCode": errorCode,
+            "errorMessage": errorMessage,
+            "details": details
+        }
+
+
+''' HELPER FUNCTIONS '''
+
+
+def map_scim(clientData):
+    try:
+        clientData = json.loads(clientData)
+    except Exception:
+        pass
+    if type(clientData) != dict:
+        raise Exception('Provided client data is not JSON compatible')
+
+    scim_extension = INPUT_SCIM_EXTENSION_KEY.replace('.', '\.')
+    mapping = {
+        "active": "active",
+        "addressCountry": "addresses(val.primary && val.primary==true).[0].country",
+        "addressFormatted": "addresses(val.primary && val.primary==true).[0].formatted",
+        "addressLocailty": "addresses(val.primary && val.primary==true).[0].locality",
+        "addressPostalCode": "addresses(val.primary && val.primary==true).[0].postalCode",
+        "addressRegion": "addresses(val.primary && val.primary==true).[0].region",
+        "addressStreeetAddress": "addresses(val.primary && val.primary==true).[0].streetAddress",
+        "addressType": "addresses(val.primary && val.primary==true).[0].type",
+        "costCenter": scim_extension + ".costCenter",
+        "department": scim_extension + ".department",
+        "displayName": "displayName",
+        "division": scim_extension + ".division",
+        "email": "emails(val.primary && val.primary==true).[0].value",
+        "emailType": "emails(val.primary && val.primary==true).[0].type",
+        "employeeNumber": scim_extension + ".employeeNumber",
+        "groups": "groups(val.display).display",
+        "id": "id",
+        "externalId": "externalId",
+        "locale": "locale",
+        "manager": scim_extension + ".manager.value",
+        "nameFormatted": "name.formatted",
+        "nameFamilyName": "name.familyName",
+        "nameGivenName": "name.givenName",
+        "nameHonorificPrefix": "name.honorificPrefix",
+        "nameHonorificSuffix": "name.honorificSuffix",
+        "nameMiddleName": "name.middleName",
+        "nickName": "nickName",
+        "organization": scim_extension + ".organization",
+        "password": "password",
+        "photo": "photos(val.type && val.type=='photo').[0].value",
+        "preferredLanguage": "preferredLanguage",
+        "profileUrl": "profileUrl",
+        "thumbnnail": "photos(val.type && val.type=='thumbnail').[0].value",
+        "timezone": "timezone",
+        "title": "title",
+        "userName": "userName",
+        "userType": "userType",
+    }
```

Comment:
do we really need all this? see when map_scim() is used in the integration and what are what are we doing with its returned value

Reply:
Removed unwanted Keys.